program-runtime/src/compute_budget.rs

```diff
@@ -10,6 +10,12 @@ use {
     },
 };
 
+/// The total accounts data a transaction can load is limited to 64MiB to not break
+/// anyone in Mainnet-beta today. It can be set by set_accounts_data_size_limit instruction
+pub const MAX_ACCOUNTS_DATA_SIZE_BYTES: usize = 64 * 1024 * 1024;
+/// default data size to allow one maximum sized account
+const DEFAULT_ACCOUNTS_DATA_SIZE_BYTES: usize = 10 * 1024 * 1024;
```

Comment:
Not sure if `DEFAULT_ACCOUNTS_DATA_SIZE_BYTES` is necessary, putting it out for debate.

From what I can see, it depends on how fees are charged for space. If it is going to be "pay for bytes exceed _xyz_", then we will need DEFAULT as _xyz_. Otherwise if it's "pay _abc_ per byte", then the default can just be the `MAX_ACCOUNTS_DATA_SIZE_BYTES`, user set more accurate data limit to save cost.

I prefer second option. Any thoughts are welcome